Parsers/ASimRegistryEvent/test/VMwareCarbonBlackCloud_vimRegistryEvent_SchemaTest.csv

```diff
@@ -0,0 +1,29 @@
+﻿Result
+"(1) Warning: Missing recommended field [ActorUserId]"
+"(1) Warning: Missing recommended field [RegistryPreviousKey]"
+"(1) Warning: Missing recommended field [RegistryPreviousValueData]"
+"(1) Warning: Missing recommended field [RegistryPreviousValueType]"
+"(1) Warning: Missing recommended field [RegistryPreviousValue]"
+"(1) Warning: Missing recommended field [RegistryValueData]"
+"(1) Warning: Missing recommended field [RegistryValueType]"
+"(1) Warning: Missing recommended field [RegistryValue]"
```

Comment:
we are missing all Registry specific recommended values, can we re-check

Reply:
We double-checked but we don't have the above fields related to the registry in VMware Carbon Black Cloud.